chatbot/model_services/builds/src/llamacpp_utils.py

```diff
@@ -0,0 +1,31 @@
+from llama_cpp import Llama
```

Comment:
I'm not sure we want to have a copy of this in every model service directory. Is there another way to manage it so this can be a share file among all the model services in the repo?

Reply:
Can we make the top directory the build context and have podman ignore the `models/` so that it does not bring in any of those larger files? That way we can still share the utils files among the different model services?

Reply:
we can use a symlink from the root of the repo src